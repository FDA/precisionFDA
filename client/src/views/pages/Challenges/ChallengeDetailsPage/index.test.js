import React from 'react'
import { shallow } from 'enzyme'

import { ChallengeDetailsPage } from '.'


describe('ChallengeDetailsPage test', () => {
  it('should render', () => {
    const wrapper = shallow(<ChallengeDetailsPage loadChallenge={() => {}} />)
    expect(wrapper).toMatchSnapshot()
  })
})
