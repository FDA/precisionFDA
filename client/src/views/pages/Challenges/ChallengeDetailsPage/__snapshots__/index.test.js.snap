// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChallengeDetailsPage test should render 1`] = `
<PublicLayout>
  <div
    className="text-center"
  >
    <Loader />
  </div>
</PublicLayout>
`;
